PoGo.NecroBot.CLI/CommandLineUtility/Arguments.cs

```diff
@@ -0,0 +1,108 @@
+﻿#region using directives
+
+using System.Collections.Generic;
+using System.Collections.Specialized;
+using System.Text.RegularExpressions;
+
+#endregion
+
+namespace PoGo.NecroBot.CLI.CommandLineUtility
+{
+    /// <summary>
+    ///     Arguments class
+    /// </summary>
+    public class Arguments
+    {
+        // Variables
```

Comment:
there is a library support this command line parse work very well and strong type code.